nixos/modules/services/backup/duplicati.nix

```diff
@@ -10,6 +10,15 @@ in
     services.duplicati = {
       enable = mkEnableOption (lib.mdDoc "Duplicati");
 
+      package = mkOption {
```

Comment:
`package` option can be simplified like this: https://github.com/NixOS/nixpkgs/blob/6bdf417578919d9d1ce7592b6cf5afa0a8237500/nixos/modules/services/x11/picom.nix#L64